### PR TITLE
Clean UG & DG

### PR DESCRIPTION
**Pull request by leezhixuan**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -130,8 +130,8 @@ Here are the other classes in `Logic` (omitted from the class diagram above) tha
...
 
 How the parsing works:
-* When called upon to parse a user command, the `AddressBookParser` class creates an `XYZCommandParser` (`XYZ` is a placeholder for the specific command name e.g., `AddCommandParser`) which uses the other classes shown above to parse the user command and create a `XYZCommand` object (e.g., `AddCommand`) which the `AddressBookParser` returns back as a `Command` object.
-* All `XYZCommandParser` classes (e.g., `AddCommandParser`, `DeleteCommandParser`, ...) inherit from the `Parser` interface so that they can be treated similarly where possible e.g, during testing.
+* When called upon to parse a user command, the `AddressBookParser` class creates an `XYZCommandParser` (`XYZ` is a placeholder for the specific command name e.g., `AddCommandParser`) which uses the other classes shown above to parse the user command and create an `XYZCommand` object (e.g., `AddCommand`) which the `AddressBookParser` returns back as a `Command` object.
+* All `XYZCommandParser` classes (e.g., `AddCommandParser`, `DeleteCommandParser`, ...) inherit the `Parser` interface so that they can be treated similarly where possible e.g, during testing.
```
Comment (Justinhoejj) [inline docs/DeveloperGuide.md:134]: Should still be inherit from I believe.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -242,15 +242,15 @@ Finds all contacts who match **ANY** name and tag keywords provided.
...
+* `findAny` is case-insensitive for names. e.g. `findAny n/hans` will match contacts named `Hans`.
 * `findAny` is case-insensitive for tags by default. e.g. `findAny t/friend` will match a contact with tags `Friend` or `friend`.
   * Adding the `c/` flag **after** the command word will make tag matching case-sensitive.  
     e.g. `findAny c/ t/friend` will match a contact with tag `friend` but not `Friend`.
 * The order of the keywords does not matter. e.g. `findAny n/Hans t/football` will return the same result as `findAny t/football n/Hans`.
 * Only full words will be matched for names and tags.
   * `findAny n/an` will not return `Hans`, but will return `Jing An`.
-* Contacts matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `findAny n/Hans t/Friend` will return all contacts with names containing the word `Hans` or who are tagged with `Friend`.
+* Only contacts that matched at least one of the keywords provided will be returned as search results (i.e., `OR` search)
```
Comment (Justinhoejj) [inline docs/UserGuide.md:252]: match instead of matched?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -89,7 +89,7 @@ It sorts all birthdays amongst all contacts in CONNECTIONS **chronologically**,
 
 ### Viewing help : `help`
 
-Shows help message explaining each command.
```
Comment (Justinhoejj) [inline docs/UserGuide.md:92]: I think this would have sufficed.

**Review by Justinhoejj — APPROVED** [review]
Some comments otherwise lgtm

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Jiwei99) [reply docs/DeveloperGuide.md:134]: Same

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -363,22 +363,22 @@ Step 3. CONNECTIONS displays a list of available commands in `ResultDisplay`.
...
 The list of birthdays is generated in the `ModelManager`, which implements the following functions:
 * `ModelManager#getBirthdayReminderList` returns an `ObservableList<Person> birthdayReminders` that is ordered according to upcoming birthdays.
 
 Given below is an example usage scenario and how the birthday reminders mechanism behaves at each step.
 
-Step 1. On app startup sort contacts with birthday by birth month and day only. Contacts with birthdays that fall on the present day are coloured green while birthdays that are within one week are coloured blue. Additionally, a customised message with each contact's phone number and age is displayed in the birthday reminder list.
+Step 1. On app startup sort contacts with a birthday by birth month and day only. Contacts with birthdays that fall on the present day are coloured green while birthdays that are within one week are coloured blue. Additionally, a customised message with each contact's phone number and age will be displayed in the birthday reminder list.
```
Comment (Jiwei99) [inline docs/DeveloperGuide.md:379]: Will `On app startup, contacts with birthday are sorted by birth month and day.` be better?

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -410,7 +410,7 @@ Step 2. The `FilteredList` in `Model` is updated.
 Step 3. The UI is updated to reflect this new state.  
 Step 4. The user provides a series of prefixes to `mailingList` to pick the fields. If no arguments are provided, default selectors are used.  
 Step 5. These `Prefix` arguments are stored in `Model`.  
-Step 6. The user is prompted to pick the name and download location of their generated CSV file.  
+Step 6. The user is prompted to pick the name and the download location of their generated CSV file.
```
Comment (Jiwei99) [inline docs/DeveloperGuide.md:413]: I think the original is fine

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -422,19 +422,19 @@ Step 10. The headers and rows are written to the CSV file that is specified by t
 
 #### Design considerations:
 * Arguments for the command should follow the standard used in other parts of the software.
-* Balancing between simplicity of use when no arguments are provided, and flexibility for users who might want additional information.
+* Balancing between the simplicity of use when no arguments are provided, and flexibility for users who might want additional information.
```
Comment (Jiwei99) [inline docs/DeveloperGuide.md:425]: Same, original is fine. Adding `the` sounds abit weird?

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -447,7 +447,7 @@ Step 3. CONNECTIONS will not display the first contact's invalid `Tag` and will
 
 #### Design considerations:
 * Arguments for the command should follow the standard used in other parts of the software.
-* Balancing between simplicity of use when no arguments are provided, and customisability for users who might want additional information.
+* Balancing between the simplicity of use when no arguments are provided, and customisability for users who might want additional information.
```
Comment (Jiwei99) [inline docs/DeveloperGuide.md:450]: Same as above

File: docs/DeveloperGuide.md [hunk 8]
```diff
@@ -519,7 +519,7 @@ Step 3. CONNECTIONS will not display the first contact's invalid `Tag` and will
 
 1.  User chooses to add a contact.
 2.  User provides the contact's details.
-3.  CONNECTIONS create an entry for that contact's details
+3.  CONNECTIONS create a contact for that contact's details
```
Comment (Jiwei99) [inline docs/DeveloperGuide.md:522]: Will `... create a contact entry ...` be better?

File: docs/DeveloperGuide.md [hunk 9]
```diff
@@ -914,10 +914,10 @@ testers are expected to do more *exploratory* testing.
 
 1. Initial launch
 
-   1. Download the jar file and copy into an empty folder.
+   1. Download the jar file and copy it into an empty folder.
 
-   1. Launch the jar file using the `java -jar connections.jar` in powershell for Windows users and terminal for Linux user. Double-click the jar file as last resort.<br>
-      Expected: Shows the GUI with a set of sample contacts. The birthday reminder list shows a birthday reminder message for each contact with non-empty birthday field. 
+   1. Launch the jar file using the `java -jar connections.jar` in powershell for Windows users and terminal for Linux users. Double-click the jar file as last resort.<br>
```
Comment (Jiwei99) [inline docs/DeveloperGuide.md:919]: `... terminal for macOS and Linux users`

File: docs/UserGuide.md [hunk 10]
```diff
@@ -113,15 +113,15 @@ Notes:
...
+Adds a contact to CONNECTIONS.
 
 #### Format:
 * `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [b/BIRTHDAY] [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">
 
 :bulb: **Tip:**
-A contact can have any number of tags (including 0).
+A contact can have a non-negative number of tags (including 0).
```
Comment (Jiwei99) [inline docs/UserGuide.md:124]: Not sure how you can have negative number of contacts but sure haha

**Review by Jiwei99 — APPROVED** [review]
Some minor comments. Otherwise lgtm

**Pull request merged**